resources/js/screens/Users/Users.tsx

```diff
@@ -109,21 +115,65 @@ const activityItems = [
     date: "2 weeks ago",
     dateTime: "2023-01-09T08:45",
   },
-];
+] as const;
 
 export const Users = () => {
+  const { pushToast } = useToastStore();
+  const queryClient = useQueryClient();
+
+  const { data: users, isLoading: isLoadingUsers } = useQuery({
+    ...getUsersQuery(),
+    select: (users) =>
+      users.map((user, idx) => {
+        const selectedItem =
+          activityItems[idx % activityItems.length] ?? activityItems[0];
```

Comment:
what is this doing exactly?

Reply:
a way to show how the select works (?. It takes the users data for the tailwindUI table columns. I'm only getting `name` and `email` from the current backend and filling them with the hardcoded data.
![Screenshot 2023-11-16 at 16 20 44](https://github.com/Light-it-labs/laravel/assets/90731332/dfcd187b-7613-4194-a5d3-81e0e8259929)